workflow/signal/signal.go

```diff
@@ -45,6 +45,13 @@ func SignalContainer(restConfig *rest.Config, pod *corev1.Pod, container string,
 
 func ExecPodContainerAndGetOutput(restConfig *rest.Config, namespace string, pod string, container string, command ...string) error {
 	x, err := common.ExecPodContainer(restConfig, namespace, pod, container, true, true, command...)
+	log.
+		WithField("namespace", namespace).
+		WithField("pod", pod).
+		WithField("container", container).
+		WithField("command", command).
+		WithError(err).
+		Info("exec command")
```

Comment:
```suggestion
		Debug("exec command")
```

Reply:
> Do we need this if #10523 works?

I think it is needed, it can provide a complete error context.

Reply:
fixed, PTAL. 😄